queries.yaml

```diff
@@ -168,3 +168,7 @@ post_process:
     source_layer: landuse
     label_property_name: label_position
     label_property_value: "yes"
+  TileStache.Goodies.VecTiles.transform.generate_label_features:
```

Comment:
Added some prs to make the conversion:

* https://github.com/mapzen/tilequeue/pull/32
* https://github.com/mapzen/vector-datasource/pull/275